### DISCUSSION TITLE
Tags are not parsing in Cards display

### DISCUSSION THREAD

**Discussion by linahin** [microsoft-search/pnp-modern-search#2993] [General]
Im using Tags from metatada already linked to a managed property. How can I make it only to show the actual value not the whole path. Please help!

**Comment by kasperbolarsen** [comment]
Can you show us the mapping? It sounds like you might have have an error in the mapping.

**Reply by linahin** [reply]
![Capture](https://github.com/microsoft-search/pnp-modern-search/assets/46541292/785197f4-fea9-4121-a327-28d11ea827b3)

**Reply by kasperbolarsen** [reply]
Nope, they look just fine, no problem there. Which of the fields are you using in the Card display and how are they defined in the "Manage Card Fields"?

**Reply by linahin** [reply]
Im using all of them since each one is a metadata value - the idea will will be to see each of the tags that the files has. Isnt that in the Tags field it displays all the tags in the card? 
![CaptureII](https://github.com/microsoft-search/pnp-modern-search/assets/46541292/86b6f209-5742-4a44-a4c0-3c11ba31eeff)

**Reply by linahin** [reply]
I need to only show the values
![CaptureIII](https://github.com/microsoft-search/pnp-modern-search/assets/46541292/9e485771-ce6c-4be5-b55c-c4a0adf98ef4)

**Reply by linahin** [reply]
@kasperbolarsen any help is appreciated :)
